<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>125</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>526</width>
    <height>141</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #ffffff;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="item_image">
       <property name="minimumSize">
        <size>
         <width>183</width>
         <height>115</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>183</width>
         <height>115</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ffffff;
text-align: center;
background:blue;</string>
       </property>
       <property name="text">
        <string>Ảnh</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="item_name">
         <property name="styleSheet">
          <string notr="true">color: black;
font-size: 16pt;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Tên sản phẩm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="item_quantity">
         <property name="minimumSize">
          <size>
           <width>307</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: gray;
font-style: italic;</string>
         </property>
         <property name="text">
          <string>Số lượng:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="item_price">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(198, 0, 0);
font-size: 18pt;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Giá tiền</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background: red;
color: #ffffff;
min-width: 50px;
max-width: 50px;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>Xoá</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
